docs/parcels.md

```diff
@@ -0,0 +1,149 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be mounted manually 
+by an application. Parcels use similar methodology as applications but are mounted by a manual function call rather than the activity function.
+A parcel can be as large as an application or as small as an component and written in 
+any language as long as it exports the correct lifecycle events. In a single-spa world, your SPA contains 
+many registered applications and potentially many parcels. Typically we recommend you mount a parcel within 
+the context of an application because the parcel will be unmounted with the application.
+
+## Parcel configuration
+A parcel is just an object with 3 or 4 functions on it, it can be loaded syncronously or asyncronously.
```

Comment:
this is two sentences mashed into one run-on sentence. I think that putting the word "it" twice in a row makes it a little hard to read.

> A parcel is just an object with 3 or 4 functions on it. When mounting a parcel, you can provide either the object itself or a loading function that asynchronously downloads the parcel object.